src/page/courier/TasksPage.js

```diff
@@ -106,13 +107,13 @@ class TasksPage extends Component {
         { cancelable: false }
       )
     })
-
   }
 
   onWebSocketConnect() {
     this.props.navigation.setParams({ connected: true })
```

Comment:
This is not sufficient: when entering this screen, as the WebSocket should already be connected, this method is never called, and the "wifi" icon is never green.